view/frontend/templates/js/replacejs.phtml

```diff
@@ -2321,12 +2321,12 @@ if (!$block->isSaveCartInSections()) { ?>
     });
 
     if (trim(location.pathname, '/') === 'checkout/cart') {
+        require(['domReady!'], function() {
         require([
         'jquery',
-        'Magento_Customer/js/customer-data',
         'Magento_Checkout/js/model/quote',
         'domReady!'
```

Comment:
This means we do not need this second `domReady!`, right?

Reply:
Correct, thank you